BenchPress/Helpers/Azure/KeyVault.psm1

```diff
@@ -209,25 +215,26 @@ function Get-KeyVaultCertificate {
 .DESCRIPTION
   Helper function for KeyVault Certificates
 
-.PARAMETER name
+.PARAMETER Name
   The name of the KeyVault Certificate
 
-.PARAMETER keyVaultName
+.PARAMETER KeyVaultName
   The name of the KeyVault
 
 .EXAMPLE
-  Get-KeyVaultCertificateExists -name "samplecertificate" -keyVaultName "kvbenchpresstest"
+  Get-KeyVaultCertificate-Name "samplecertificate" -KeyVaultName "kvbenchpresstest"
```

Comment:
`KeyVaultCertificate` -> `KeyVaultCertificateExist` and missing space before `-Name`